Client/Frontend/Strings.swift

```diff
@@ -3772,20 +3772,20 @@ extension String {
             value: "Show products recommended by %@",
             comment: "Label of the switch from settings card displayed in the shopping review quality bottom sheet. The placeholder will be replaced with the app name.")
         public static let SettingsCardTurnOffButton = MZLocalizedString(
-            key: "Shopping.SettingsCard.TurnOff.Button.v120",
+            key: "Shopping.SettingsCard.TurnOffButton.Title.v120",
             tableName: "Shopping",
             value: "Turn Off Review Checker",
             comment: "Label of the button from settings card displayed in the shopping review quality bottom sheet.")
         public static let SettingsCardExpandedAccessibilityLabel = MZLocalizedString(
             key: "Shopping.SettingsCard.Expanded.AccessibilityLabel.v120",
             tableName: "Shopping",
-            value: "Settings Card Expanded",
-            comment: "Accessibility label for the down chevron, from Settings Card View displayed in the shopping review quality bottom sheet.")
+            value: "Expand Settings Card",
+            comment: "Accessibility label for the down chevron icon used to expand or show the details of the Settings Card within the shopping product review bottom sheet.")
         public static let SettingsCardCollapsedAccessibilityLabel = MZLocalizedString(
             key: "Shopping.SettingsCard.Collapsed.AccessibilityLabel.v120",
             tableName: "Shopping",
-            value: "Settings Card Collapsed",
-            comment: "Accessibility label for the up chevron, from Settings Card View displayed in the shopping review quality bottom sheet.")
+            value: "Collapse Settings Card",
```

Comment:
This is the result of this change:
<img width="1385" alt="Screenshot 2023-10-23 at 10 38 37" src="https://github.com/mozilla-mobile/firefox-ios/assets/4530/220392f3-e4bb-4b55-b90c-c9ded7e15da8">
Now a collapsed card will tell the user the button will collapse the card - which is incorrect.

Reply:
Makes sense. So it should say "Expand card ..."? For me it still doesn't sound 100% right.

Reply:
I'd check with @ewachowiak about this.

Reply:
Updated for now to reflect the opposite state. Will check with @ewachowiak .

Reply:
Yes, it makes sense to say "Expand" when the card is closed and "Collapse" when the card is open to indicate what will happen when they tap the chevron.